contracts/mainnet/connectors/uniswapV3/main.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3.
+ * @dev Decentralized Exchange.
+ */
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+abstract contract UniswapResolver is Helpers, Events {
+    /**
+     * @dev Mint New Position
+     * @param params: parameter for mint.
+     * @param getId: ID to retrieve amtA.
+     * @param setId: ID stores the amount of LP token.
+     */
+    function mint(
+        MintParams memory params,
```

Comment:
@cryptoDev222 can we pass as normal args? Instead of struct? If possible only.

Reply:
If we use normal args, it causes stack too deep error.